pkg/api/convertertoapi.go

```diff
@@ -695,6 +695,7 @@ func convertVLabsKubernetesConfig(vlabs *vlabs.KubernetesConfig, api *Kubernetes
 	api.CustomCcmImage = vlabs.CustomCcmImage
 	api.UseCloudControllerManager = vlabs.UseCloudControllerManager
 	api.CustomWindowsPackageURL = vlabs.CustomWindowsPackageURL
+	api.WindowsNodeBinariesURL = vlabs.WindowsNodeBinariesURL
```

Comment:
Same as previous comment

Reply:
There is not a precedent for testing this particular set of settings.  I gave it a go but it feels a little strange as it's just copying values from one place to another.

I tried two different ways to test.  I reverted one of the tries and went a different route.  Can rebase those commits away unless you perfer the first try.